### DISCUSSION TITLE
Screenshot action ~ missing description of usage in various help texts (including wiki)

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1161] [Q&A] [answered]
In process of improving path panel helptext, I paused upon hitting the screenshot path, primarily and TBH being a function I'd not used or checked out before. Ergo, I asked myself...  "How does one enable the screenshot functionality in amiberry?" ....

- nothing on the wiki refers an answer to this
- likewise inbuilt helptext offers no clue

After a glimpse of master tree, I found 'Screenshot (file)' as an action in custom controls, and therein lies the answer ~ you have to map the screenshot function to a control input.... and I thought "dang, that's a tad obscure with no pointers" ... ;) 

It likely maps to printscreen _ sysreq on this kbd by default (?..I'll check later) ~ I've no idea on 'user demand' for this action, however if user desires such, it's not exactly intuitive wrt how to do this....

.... wrt the wiki, perhaps it might be expanded by adding the above Q&A...to the FAQ section?

Following on, if we were to presume that a typical user want, would be to map a controller button to screenshot (file), then the dropdown menu ordering in the Custom Controls panel, probably isn't ordered in a way to proffer that presumption...ie;

![example](https://imgur.com/2Wg33y2.png)

For mine, 'Screenshot (file)' should be top of that listing, representing an option that most users would want to immediately use (and if I'm still sane, mapping a screenshot button to a multi-button controller would be high on users' want to do list?) Likewise, any other highly likely action targets should be situated towards to top of the list ...a form of usage intuition ... =)

What do you think?

TIA

**Comment by midwan** [comment, marked as answer]
The screenshots are captured from Amiberry directly, not through the system, and saved as PNGs to the configured path.
The savestate mechanism uses the same thing also, to show you thumbnails of each saved state you create. It's true that it's not mentioned in the wiki or Help, mostly because I forgot to add it :)

The order of events shown in the dropdown is located in `amiberry_input.cpp`, and newer entries usually go at the bottom, that's why they show like that there also. In combination with the GUI toolkit limitations, it's not very pretty... Some events can be mapped to hotkeys as well, from the Misc panel. Though since space is limited there, only those considered more essential are added. Nobody has asked about Screenshots so far, so it never became important enough to make it more visible.

**Comment by giantclambake** [comment]
Okay, understood  ...that being the case, might be best to let sleeping dogs lay ~ if someone ever queries for screenshot functionality, then there'd be a tacit reason to revisit this.  

Thanks!
